Fix Javadoc warnings

### Describe the bug

There are javadoc warnings

### Expected behavior

_No response_

### Minimal reproducible example

_No response_

### Add-on Version

1.0.0

### Vaadin Version

-

### Additional information

```
[WARNING] Javadoc Warnings
[WARNING] J:\fc\addons\ChatAssistant\src\main\java\com\flowingcode\vaadin\addons\chatassistant\ChatAssistant.java:327: warning: @param argument "rowIndex" is not a parameter name.
[WARNING] * @param rowIndex
[WARNING] ^

--- a/pom.xml
+++ b/pom.xml
@@ -314,7 +314,7 @@
                         <configuration>
                             <quiet>true</quiet>
                             <doclint>none</doclint>
-                            <failOnWarnings>false</failOnWarnings>
+                            <failOnWarnings>true</failOnWarnings>
                             <links>
                                 <link>https://javadoc.io/doc/com.vaadin/vaadin-platform-javadoc/${vaadin.version}</link>
                             </links>

--- a/src/main/java/com/flowingcode/vaadin/addons/chatassistant/ChatAssistant.java
+++ b/src/main/java/com/flowingcode/vaadin/addons/chatassistant/ChatAssistant.java
@@ -318,13 +318,13 @@ public class ChatAssistant extends Div {
   }
   
   /**
-   * Scrolls to the given row index. Scrolls so that the element is shown at
+   * Scrolls to the given position. Scrolls so that the element is shown at
    * the start of the visible area whenever possible.
    * <p>
    * If the index parameter exceeds current item set size the grid will scroll
    * to the end.
    *
-   * @param rowIndex
+   * @param position
    *            zero based index of the item to scroll to in the current view.
    */
   public void scrollToIndex(int position) {

